src/routes/registerToken.tsx

```diff
@@ -52,13 +52,17 @@ export default class RegisterToken extends React.Component<RegisterTokenProps> {
     return (
       <AppLayout section="registerToken">
         <h1 style={{ marginBottom: 16, marginTop: 32 }}>Registered tokens:</h1>
-        <List
-          itemLayout="vertical"
-          size="small"
-          className="tokens-list"
-          dataSource={tokensStore.ready ? tokensStore.list : undefined}
-          renderItem={item => <Item item={item} />}
-        />
+        {tokensStore.ready ? (
+          <List
+            itemLayout="vertical"
+            size="small"
+            className="tokens-list"
+            dataSource={tokensStore.ready ? tokensStore.list : undefined}
+            renderItem={item => <Item item={item} />}
+          />
+        ) : (
+          <Spin />
```

Comment:
Personally, I would add some message here. 
Something like "Loading a list of tokens, please wait..."